superbench/benchmarks/micro_benchmarks/gpu_copy_performance/gpu_copy.cu

```diff
@@ -313,6 +318,22 @@ int SetGpu(int gpu_id) {
     return 0;
 }
 
+#if defined(__HIP_PLATFORM_AMD__)
+cudaError_t GpuMallocDataBuf(uint8_t **ptr, uint64_t size, bool use_fine_grained) {
```

Comment:
Why not remove bool use_fine_grained directly?

Reply:
See above